dcs/mission.py

```diff
@@ -2,6 +2,7 @@
 The mission module is the entry point to all pydcs functions.
 """
 import copy
+
```

Comment:
why the newline?

Reply:
Not sure, I ran flake8 and fixed until it stopped complaining. ;)